### DISCUSSION TITLE
Feature Request: Support custom image display size

### DISCUSSION THREAD

**Discussion by ichxxx** [notaapp/talk#256] [Feature Requests]
I found that the image display on nota seems to have only a fixed size, and it's too small for me.

It would be nice to be able to support custom image display size, so that I don't have to click the image to zoom in every time to browse the information.

Thank you.

**Comment by astoilkov** [comment]
Hi. We have a setting for that. You can find it in Preferences (⌘,) → Editor → Image Height. Will that work for you?

**Reply by ichxxx** [reply]
It works, thank you! But I think it would be better to the feature the title says, because I might want to have different display sizes for different images, instead of a fixed size.

**Reply by astoilkov** [reply]
I agree. I will look into how we can do this. Thanks!
